docs/desktop/file-shredder.md

```diff
@@ -0,0 +1,53 @@
+---
+title: File Shredder 
+author: Christine Belzie
+contributors:  
+---
+ 
+## Introduction
+
+Need to remove a postcard or a PDF file that contains information like your birthday? Check out File Shredder. It is an application that permanently removes sensitive information online.
+
+## Assumptions
+
+This guide assumes you have the following:
+
+- Rocky Linux
+- Flatpak
+- FlatHub
+
+## Installation Process
+
+![Screenshot of the File Shredder app page on FlatHub, showing the blue install button being highlighted by a red rectangle](images/01_file-shredder.png)
+
+1. Go to [Flathub's website](https://flathub.org), type "File Shredder" in the search bar, and click on **Install**
+
+![manual install script and run script](images/file-shredder_install.png)
+
```

Comment:
The screenshot above needs to be indented as it comes *after* the numbered list item. I'll fix it here.

Reply:
fixed and ready to merge. Will do a quick edit after.